content/companies/06-gojek-tech.md

```diff
@@ -16,14 +15,20 @@ index: 6
+2. **Pair Programming / Live Coding:** This round is about candidate pairing with one of Gojek's engineers to solve new set of problems related to the submitted assignment. These problems are an extension to the original assigment. The focus of this round is to see if the candidate is very vocal about their thought process and also to see how efficiently the new problem can be solved/coded. Basic concepts on programming paradigms are explored as well. Discussions on the choice of tech stack, design patterns etc are a likelyhood in this round.
+3. **Tech 1:** This round focusses on system design. This is some sort of a white board round where the interviewer gives a scenario of a real life product and the candidate would need to design a the application with scale in mind. Constraints are presented as and when you are explaning your solution and you will need to work around these constraints. This is usually a fun round and you are expected to be as creative as possible to design a cool production grade system of the product.
+4. **Tech 2:** In this round candidate will be interviewed on their past experiences, projects they have worked on or lead on, challenges that were faced. A discussion on topics such as problems faced and how they were solved, decisions taken, outcomes, aftermath, retro etc do happen in this round. This is where you can go into depths of detail on certain experiences in your career and how you overcame those moments. Following this round there may be a Tech 3 round, which is very subjective on the candidate profile.
+5. **Bar Raiser:** In this round a skilled evaluator who plays a crucial role in the company's hiring process interviews the candidate. This round represents the final hurdle for incoming candidates. They run the last interview in the process, making the final judgment on applicants.
+6. **Negotiaton:** This round focusses on compensation discussion.
 
 ## Appendix
 - **Rounds(approx)** - Total number of rounds. This an approximate number. Can vary depending on what role you're applying for.
 - **Scripted Questions** - Describes if the company asks questions which are kind of direct bookish questions or can be easily found on internet infact on the first result page on google 😛. Example: Write a function to print first 100 fibonnaci numbers, How does event loop works, write a polyfil for X etc.
-- **DS and Algo** - Describes if the company asks DS and algo questions which which are kind of direct bookish questions about data structure and algorithms instead of a real life problem.
\ No newline at end of file
+- **DS and Algo** - Describes if the company asks DS and algo questions which which are kind of direct bookish questions about data structure and algorithms instead of a real life problem.
+
+
+## Reporter Info
+- **Name:** Aniket Rao
+- **Twitter:** [@aniket_rao](https://twitter.com/aniket_rao)
+- **Role/Level:** Product Engineer
```

Comment:
Is this your current role? Or the role/level this process belongs to?

Reply:
Is my current role and also the most frequently hired role and also the current process.